src/Commands/FullIntake.cpp

```diff
@@ -0,0 +1,14 @@
+#include "FullIntake.h"
+#include "SetArms.h"
+#include "IntakeOn.h"
+#include "ShooterControl.h"
+#include "RollerOn.h"
+
+FullIntake::FullIntake(double speed)
+: mSpeed(speed)
+{
+  AddSequential(new SetArms(Intake::kDown));
```

Comment:
Should Intake be include? Since we use Intake::Down. It is being included in IntakeOn however...